sql/i4_invalid_indexes.sql

```diff
@@ -0,0 +1,25 @@
+-- Lisf of invalid Indexes 
+
+-- Use it to see invalid indexes list
+
+-- This query doesn't need any additional extensions to be installed
+-- (except plpgsql), and doesn't create anything (like views or smth)
+-- -- so feel free to use it in your clouds (Heroku, AWS RDS, etc)
+
+-- It also does't do anything except reading system catalogs and
+-- printing NOTICEs, so you can easily run it on your
```

Comment:
I don't see any printed NOTICEs, this query is a simple SELECT returning some set of records. Not NOTICEs are to be produced.